packages/inter-protocol/src/proposals/upgrade-vaults.js

```diff
@@ -181,7 +181,7 @@ export const getManifestForUpgradeVaults = async (
       installation: {
         produce: { VaultFactory: true },
       },
-      instance: { consume: { auctioneer: uV } },
+      instance: { consume: true },
```

Comment:
Granting more authority is often suspicious... but all the instances here are intended to be widely shared anyway. So this is fine.